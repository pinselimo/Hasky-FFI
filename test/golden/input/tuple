tuple :: (CInt, String)
tuple = (1 :: CInt, "a")

