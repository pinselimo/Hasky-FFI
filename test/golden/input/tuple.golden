tuple :: (CInt, String) -> (CInt, String)
tuple = id

